tests/websocket.spec.js

```diff
@@ -1,38 +1,45 @@
-var assert = require('assert');
-var nock = require('nock');
+const assert = require('assert');
+const nock = require('nock');
 
-var Gdax = require('../index.js');
+const Gdax = require('../index.js');
 
-var key = 'key';
-var secret = 'secret';
-var passphrase = 'passphrase';
+const key = 'key';
+const secret = 'secret';
+const passphrase = 'passphrase';
 
-var testserver = require('./lib/ws_testserver');
-var port = 56632;
+const testserver = require('./lib/ws_testserver');
+let port = 56632;
 
-suite('WebsocketClient');
+suite('WebsocketClient', () => {
 
-describe('WebsocketClient', function() {
   test('connects to specified server', function(done) {
-    var server = testserver(++port, function() {
-      var websocketClient = new Gdax.WebsocketClient(['BTC-EUR'], 'ws://localhost:' + port);
+    const server = testserver(++port, function() {
+      const websocketClient = new Gdax.WebsocketClient(
+        ['BTC-EUR'],
+        'ws://localhost:' + port
+      );
       websocketClient.on('open', function() {
         server.close();
         done();
       });
     });
   });
 
-  test('subscribes to the default product (BTC-USD) if undefined', function(done) {
-    var server = testserver(++port, function() {
-      var websocketClient = new Gdax.WebsocketClient(null, 'ws://localhost:' + port);
+  test('subscribes to the default product (BTC-USD) if undefined', function(
```

Comment:
All of these should be switched to arrow functions (eslint can also take care of that — only named functions should still be allowed).